docs/fs.md

```diff
@@ -43,13 +52,21 @@ of older kernels, with additional patches, are not supported.
 ## How to share directories with cloud-hypervisor
 
 ### Start the VM
-Once the daemon is running, the option `--fs` from __cloud-hypervisor__ needs to be used.
 
-Direct kernel boot is the preferred option, but we can boot from an EFI cloud image if it contains a recent enough kernel.
+Once the daemon is running, the option `--fs` from Cloud Hypervisor needs
+to be used.
 
-Because _vhost-user_ expects a dedicated process (__virtiofsd__ in this case) to be able to access the guest RAM to communicate through the _virtqueues_ with the driver running in the guest, `--memory` option needs to be slightly modified. It must specify `shared=on` to share the memory pages so that an external process can access them.
+Direct kernel boot is the preferred option, but we can boot from an EFI cloud
+image if it contains a recent enough kernel.
 
-Assuming you have `focal-server-cloudimg-amd64.raw` and `vmlinux` on your system, here is the __cloud-hypervisor__ command you need to run:
+Because _vhost-user_ expects a dedicated process (__virtiofsd__ in this case)
+to be able to access the guest RAM to communicate through the _virtqueues_ with
+the driver running in the guest, `--memory` option needs to be slightly
+modified. It must specify `shared=on` to share the memory pages so that an
+external process can access them.
```

Comment:
This whole paragraph can be simplified to: "Correct functioning of `--fs` requires `--memory shared=on` to facilitate interprocess memory sharing."